charts/invenio/templates/vocabularies-configmap.yaml

```diff
@@ -0,0 +1,10 @@
+{{- with .Values.invenio.vocabularies }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: invenio-vocabularies
+  labels:
+    {{- include "invenio.labels" . | nindent 4 }}
```

Comment:
Please verify by running `helm template` if `include "invenio.labels" .` works or if we need to use `include "invenio.labels" $` instead (since `with` changes the scope). I'm not sure, we need to check.

Reply:
Good catch!